compiler/optimizer/LoopVersioner.cpp

```diff
@@ -5915,7 +5912,7 @@ void TR_LoopVersioner::buildConditionalTree(
                TR::Node *storeNode = _storeTrees[indexNode->getSymbolReference()->getReferenceNumber()]->getNode();
                //int32_t exitValue = exitValue = storeNode->getFirstChild()->getSecondChild()->getInt();
 
-               loopLimit = _loopTestTree->getNode()->getSecondChild()->duplicateTree(comp());
+               loopLimit = _loopTestTree->getNode()->getSecondChild()->duplicateTree();
```

Comment:
In this bug we are passing the compilation object pointer as a boolean which determines whether to duplicate children.